consensus/src/ledger.rs

```diff
@@ -209,13 +209,31 @@ impl OperationLedgerInterface for Operation {
                     &LedgerChange {
                         balance_delta: roll_price
                             .checked_mul_u64(*roll_count)
-                            .ok_or(ConsensusError::RollOverflowError)?,
+                            .ok_or(ConsensusError::AmountOverflowError)?,
                         balance_increment: false,
                     },
                 )?;
             }
             // roll sale is handled separately with a delay
             models::OperationType::RollSell { .. } => {}
+            models::OperationType::ExecuteSC {
```

Comment:
Can we add a link to the spec here? @damip

Reply:
yes, when we merge the spec